Druid/CompiledMethod.extension.st

```diff
@@ -0,0 +1,13 @@
+Extension { #name : #CompiledMethod }
+
+{ #category : #'*Druid' }
+CompiledMethod >> numberOfArguments [
+
```

Comment:
Be careful, this method name could override  a method in Pharo and break everything...

Reply:
I checked in a Pharo image with Druid from main branch:

```smalltalk
#numberOfArguments implementors "an OrderedCollection(DRBlockClosure>>#numberOfArguments DRCogitSendMarshall>>#numberOfArguments DRJITMessageSend>>#numberOfArguments TraitMethodDescription>>#numberOfArguments VMCompiledCodeBuilder>>#numberOfArguments)"
```

so it seems isn't breaking anything but for method extensions in general I would prefix with #dr ...all methods since these kind of selectors could break core tools. What do you think?

Reply:
I don't know, I've got lost with "this", "them", "these".... Do you have an example?